dtm0/service.c

```diff
@@ -159,23 +209,27 @@ M0_INTERNAL int m0_dtm0_service_process_disconnect(struct m0_reqh_service *s,
 						   struct m0_fid *remote_srv)
 {
 	int rc;
```

Comment:
I treat this function as a non-completed.

Reply:
Yes, currently we are targeting on a "happy path".